addons/account/views/account_report.xml

```diff
@@ -30,7 +30,6 @@
             report_type="qweb-pdf"
             name="account.account_invoice_report_duplicate_main"
             file="account.report_invoice_duplicate"
-            attachment_use="False"
```

Comment:
Question: The reload from attachment feature will be missing in the next version? can you enlight us which is the rationale my friend? @qdp-odoo